lgc/patch/FragColorExport.cpp

```diff
@@ -513,6 +513,8 @@ bool LowerFragColorExport::runOnModule(Module &module) {
     lastExport = lastGenericExport;
   }
 
+  m_resUsage->inOutUsage.fs.dummyExport =
+      (m_pipelineState->getTargetInfo().getGfxIpVersion().major < 10 || m_resUsage->builtInUsage.fs.discard);
```

Comment:
Don't know how I missed this.  All of my testing is on gfx9.  I'll have to see if I can test on gfx10.  I think I have one available.